Bottom: 7c5eed134a7769c737e752eb1b51805aad9a2064
Top:    db03ccf60c2728e7de63e51d9329bfe19ea544c0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 04:19:55 +0000

HMM - THIS APPEARS TO FIX BROKEN DRILLED HOLES IN THE PCB CORE


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 1022855..7d4b3c6 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -300,7 +300,7 @@ get_contour_edge_n_round_geometry_in_step_mm (PLINE *contour, int n, double *cx,
       *cx = COORD_TO_STEP_X (PCB, contour->cx);
       *cy = COORD_TO_STEP_Y (PCB, contour->cy);
       *r = COORD_TO_MM (contour->radius);
-      *cw = (contour->Flags.orient == PLF_DIR);
+      *cw = (contour->Flags.orient != PLF_DIR);
       return;
     }
 #endif
